modules/utxo-lib/src/coins.ts

```diff
@@ -173,18 +173,12 @@ export function isZcash(network) {
 }
 
 /**
- * @param {Network} network
+ * @param {unknown} network
  * @returns {boolean} returns true iff network is any of the network stated in the argument
  */
-export const isValidNetwork = typeforce.oneOf(
-  isBitcoin,
-  isBitcoinCash,
-  isBitcoinGold,
-  isBitcoinSV,
-  isDash,
-  isLitecoin,
-  isZcash
-);
+export function isValidNetwork(network: unknown): network is Network {
+  return Object.values(networks).includes(network as Network);
+}
```

Comment:
previously, this method threw an exception if the argument was not a known network

not the method returns false

Reply:
Fwiw, it looks like `typeforce` is no longer used in this file after this change.